src/asmcnc/core_UI/sequence_alarm/alarm_manager.py

```diff
@@ -270,6 +322,8 @@ def setup_report(self):
 		except:
 			alarm_number = ""
 
+		if not self.sg_alarm: description = self.l.get_str(self.alarm_description)
+		else: description = (self.l.get_str("The N axis was overloaded during a move.")).replace("N", self.stall_axis)
 
```

Comment:
`description = description = (self.l.get_str("The N axis was overloaded during a move.")).replace("N", self.stall_axis) if sg_alarm else self.l.get_str(self.alarm_description)`

Also when checking both cases of true/false with if/else for readability do if variable else not variable rather than vice versa

Reply:
If most cases are going to be self.alarm_description, isn't is slightly more efficient to do:

description = self.l.get_str(self.alarm_description) if not self.sg_alarm else (self.l.get_str("The N axis was overloaded during a move.")).replace("N", self.stall_axis)

from a processing perspective? 

(I may be wrong) 

Also, is this the best place to put a line break: 

		description = self.l.get_str(self.alarm_description) if not self.sg_alarm \
		else (self.l.get_str("The N axis was overloaded during a move.")).replace("N", self.stall_axis)
		
?

Reply:
I don't think so as it's already checking the value of sg_alarm so it's just a matter of 1/0 but either way probably very marginal. I just came from a readability perspective as you usually expect the NOT case to be in the else. 

And yes I think that line break works well.